js/ad_manager_vast.js

```diff
@@ -1875,6 +2127,625 @@ OO.Ads.manager(function(_, $) {
       adSource.followRedirects = $(adSourceElement).attr("followRedirects");
       return adSource;
     }, this);
+
+    /**
+     * Generates a parsed VPaid ad to load.
+     * This is only required for VPAID ads
+     * @private
+     * @method Vast#_getVPaidCreative
+     * @param {XMLDocument} adXml Current ad xml
+     * @param {string} Current vast version
+     * @param {object} adLoaded The ad loaded object and metadata
+     * @return {object} Parsed vpaid's metadata ad
+     */
+    var _getVpaidCreative = _.bind(function(adXml, version, adLoaded) {
```

Comment:
Can you add some comments in the function on what is happening? Will help with maintenance down the road. :+1: